frontend/src/components/GroupView/MiddleBar/GroupViewMiddleBarTaskQueue.module.scss

```diff
@@ -30,3 +30,8 @@
   cursor: pointer;
   opacity: 0.7;
 }
+
+.TaskList {
+  max-height: 85%;
```

Comment:
why this weird percentage?

Reply:
This was so that the rendered task list does not overflow into the `People` section. I just changed its height values in a recent commit.